src/System.IO.Pipes/tests/Interop.Windows.cs

```diff
@@ -65,31 +63,8 @@ internal static bool TryGetNumberOfServerInstances(SafePipeHandle handle, out in
             return false;
         }
 
-        #endregion
-
-        #region Unix
-
-        [DllImport("libc", SetLastError = true)]
-        private static extern unsafe int gethostname(byte* name, int len);
-
-        internal static unsafe bool TryGetHostName(out string hostName)
-        {
-            const int HOST_NAME_MAX = 255; // man gethostname
-            const int ArrLength = HOST_NAME_MAX + 1;
-
-            byte* name = stackalloc byte[ArrLength];
-            int result = gethostname(name, ArrLength);
-
-            if (result == 0)
-            {
-                hostName = Marshal.PtrToStringAnsi((IntPtr)name);
-                return true;
-            }
-
-            hostName = "";
-            return false;
-        }
-
-        #endregion
+        // @todo: These are called by some Unix-specific tests. Those tests should really be split out into
+        // partial classes and included only in Unix builds.
```

Comment:
No, the goal of the PR is to get the test unblocked on AoT as quickly as possible. The todo's are just nice-to-do cleanup, but they're definitely a pri-3 issue.